BSP/STM32F413/Makefile

```diff
@@ -126,21 +125,21 @@ C_DEFS =  \
 # AS includes
 AS_INCLUDES = 
 
-# $(addsuffix /Inc, $(MODULES))	\
-$(OS_SRC_DIRS)
-
 # C includes
 C_INCLUDES :=  \
-$(TOP_LEVEL)/Config/Inc/	\
-$(BSP_DIR)/../Inc/	\
-$(BSP_DIR)/Inc/	\
-$(BSP_DIR)/STM32F4xx_HAL_Driver/Inc/	\
-$(BSP_DIR)/STM32F4xx_HAL_Driver/Inc/Legacy/	\
-$(CMSIS_DIR)/Device/ST/STM32F4xx/Include/      \
-$(CMSIS_DIR)/Core/Include/	\
-
-# Add appropriate prefix to all includes
+$(TOP_LEVEL)/Config/Inc	\
+$(addsuffix /Inc, $(MODULES))	\
```

Comment:
Why this?

Reply:
MODULES is the variable I use for all the top level folders: Apps, Tasks, Drivers, and anything else we want to include in the future; basically middleware and high level software. This adds /Inc to the end of all of them in order to include their headers.

Reply:
Since OS is structured a little differently in that all the headers are in the same place as source code, OS_SRC_DIRS stores all the OS directories that may have code in them.